app/views/layouts/_lte_admin_sidebar.html.erb

```diff
@@ -64,6 +64,11 @@
           <i class="nav-icon fa fa-plus-circle"></i> New Organization
         <% end %>
       </li>
+      <li class="nav-item <%= 'active' if current_page?(admin_broadcast_announcements_path) %>">
+        <%= link_to(admin_broadcast_announcements_path, class: "nav-link #{"active" if current_page?(admin_broadcast_announcements_path)}") do %>
+          <i class="nav-icon fa fa-circle-o"></i> Bank Announcement
+        <% end %>
+      </li>
```

Comment:
On actually trying it out,  I think it'll be more natural to have this as "Announcements" at the top level, rather than within the orgranization.   Thanks!

Reply:
Sounds good! @cielf I'll make that change now